docker/bigdl/Dockerfile

```diff
@@ -103,6 +104,7 @@ ENV RUNTIME_DRIVER_CORES            ${RUNTIME_DRIVER_CORES}
 ENV RUNTIME_DRIVER_MEMORY           ${RUNTIME_DRIVER_MEMORY}
 ENV RUNTIME_EXECUTOR_CORES          ${RUNTIME_EXECUTOR_CORES}
 ENV RUNTIME_EXECUTOR_MEMORY         ${RUNTIME_EXECUTOR_MEMORY}
+ENV RUNTIME_EXECUTOR_INSTANCES      ${RUNTIME_EXECUTOR_INSTANCES}
 ENV RUNTIME_TOTAL_EXECUTOR_CORES    ${RUNTIME_TOTAL_EXECUTOR_CORES}
```

Comment:
also delete this RUNTIME_TOTAL_EXECUTOR_CORES    
we use RUNTIME_EXECUTOR_INSTANCES to replace RUNTIME_TOTAL_EXECUTOR_CORES    
and please update the doc

Reply:
Completed, please take a look.

Reply:
Completed, please take a look.